doc/autoscale/experiment/README.md

```diff
@@ -1,26 +1,40 @@
 # Auto-scaling Experiment Design
 
+## Purpose
 
-## Environment Enviroment
+To verify the effectiveness of PaddlePaddle's fault-tolerance and auto-scaling mechanism.
 
-- Kubernetes v1.6 cluster with 133 nodes.
-- PaddleCloud with latest develop branch installed.
-- We will train the [recognize_digits](https://github.com/PaddlePaddle/cloud/tree/develop/demo/recognize_digits) model in the experiment.
+## Metrics
 
+How the effectiveness is measured.
 
-## Experiment Metric
+1. Cluster computing resource overall utilization.
+    - the higher the better.
+    - higher utilization means less resource is idle.
```

Comment:
> higher utilization means less resource is idle.

This sentence is too simple. May describe like this:

Autoscaling intended to maximize the overall cluster resource(CPU, GPU, memory) usage by ensuring resource for production level jobs/services, then fairly scale jobs that are scalable to use resource left in the cluster.